spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderTests.java

```diff
@@ -59,8 +58,8 @@
 import org.springframework.retry.policy.SimpleRetryPolicy;
 import org.springframework.retry.support.RetryTemplate;
 
-import kafka.admin.AdminUtils;
-import kafka.api.TopicMetadata;
+import static org.assertj.core.api.Assertions.assertThat;
```

Comment:
We observe the code conventions of Spring Boot ATM, so perhaps we can adjust this when we settle on a code convention set.